erpnext/accounts/report/statement_of_account/statement_of_account.py

```diff
@@ -98,7 +106,7 @@ def get_conditions(filters):
 		conditions.append("party_type=%(party_type)s")
 
 	if filters.get("party"):
-		conditions.append("party in %(party)s")
+		conditions.append("party=%(party)s")
```

Comment:
- This will break since party is a multiselect field

Reply:
@hrwX We have removed multi select field. thats why we modified this one